src/framework/global/internal/interactive.cpp

```diff
@@ -170,7 +170,7 @@ io::paths Interactive::selectMultipleDirectories(const QString& title, const io:
 
     RetVal<Val> paths = open("musescore://interactive/selectMultipleDirectories?" + params.join("&").toStdString());
     if (!paths.ret) {
-        return io::paths();
+        return selectedDirectories;
```

Comment:
Thanks!
I would also ask to get rid of this logic in the dialog.
And remove m_originDirectories from the model.
![image](https://user-images.githubusercontent.com/10116828/169217451-d1a44239-e31c-4cc6-ae48-32ff90f10821.png)

Reply:
@Eism Done! Please have a look

Reply:
thanks!